src/scroll-timeline-base.js

```diff
@@ -473,6 +473,7 @@ export function calculateRange(phase, container, target, axis, optionsInset) {
   // The view position is the scroll position of the logical starting edge
   // of the view.
   const style = getComputedStyle(container);
+  const targetBCR = target.getBoundingClientRect();
```

Comment:
getBoundingClientRect is transformed through all of the ancestor transforms applying to that element. This won't give the correct result if there are any scale transforms applied. You should use offsetWidth / offsetHeight instead of clientWidth / clientHeight as these return what we want, the element's local box size including borders according to https://drafts.csswg.org/cssom-view/#dom-htmlelement-offsetwidth

Reply:
Thanks 🙏 
I've updated the PR.